packages/headless/src/features/search/search-slice.test.ts

```diff
@@ -359,50 +349,6 @@ describe('search-slice', () => {
     });
   });
 
-  describe('should dispatch a logQueryError action', () => {
```

Comment:
Added one test in `search-action-thunk-processor` instead, which cover all these.